web/src/components/TransactionRunResult/ExecutionStep.tsx

```diff
@@ -26,7 +27,7 @@ interface IProps {
 const ExecutionStep = ({
   index,
   test: {name, trigger, id: testId},
-  testRun: {id: runId, state, testVersion} = TestRun({}),
+  testRun: {id: runId, state, testVersion, passedAssertionCount, failedAssertionCount} = TestRun({}),
```

Comment:
Looks great @xoscar I saw in your video that you needed to refresh the page to see the correct results. Is there a problem with the websocket data?

Reply:
Yep, problem has been reported to the BE team they are working on it 😄